src/vsr/replica.zig

```diff
@@ -6569,11 +6588,12 @@ pub fn ReplicaType(
             assert(self.pipeline == .cache);
 
             var op = self.commit_max + 1;
-            while (op <= self.op) : (op += 1) {
-                const op_header = self.journal.header_with_op(op).?;
-                if (!self.pipeline.cache.contains_header(op_header)) {
-                    return op;
-                }
+            const view_headers_op_max = self.view_headers.array.get(0).op;
```

Comment:
Assert that self.view_headers.command is an SV

Reply:
I like how we now don't look at the journal and use an SV as a source of truth. THis is correct, because SV _should_ include the piepeline. 

But can we additionally cross-assert that what's in our SV is consistent with what's in our journal?

Reply:
https://github.com/tigerbeetle/tigerbeetle/pull/2236/commits/36fef729e334b6fefa0e6ad20fe7f68c0367b457#diff-e95e4dc6c1aae349e798cfeed22ad01f830cce0e654e6b4b765b3e7b0c677b6aR6605-R6612